### ISSUE TITLE
Previous and next button labels are announced twice for "multiple app level Alerts"

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4578]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 3.3.2 Labels or Instructions

**Test Environment:**
- Test URL: https://clarity.design/documentation/alerts
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto to the previous or next button provided for multiple app level alerts component and observe the screen reader&#039;s spoken feedback.

**Expected result:**
Previous or next image buttons labels should be announced appropriately with the control type .

**Actual result:**
Previous and next image button labels are announced twice in the following way:
"Previous Previous, graphic, button" and "Next Next, graphic, button" respectively.

**Root cause:**
Title attribute and aria-labelledby attribute is used for the previous/next graphic icons. Values of those attributes are same, so screen reader is announcing those values.

**Comment by gkhadK** [comment]
Similar issue is observed for icon buttons provided in Buttons and Button groups components
